acm.md

```diff
@@ -0,0 +1,62 @@
+## Setup ACM with NetObserv metrics
+
+cf also [blog post](./blogs/acm/leverage-metrics-in-acm.md).
+
+This is more a quick guide for the development teams.
+
+Quick guide:
+
+1. Create 2 clusters (or more)
+2. Choose one for being the main one / hub: install ACM operator on it; Create a default MultiClusterHub
+3. In console top bar, select "all cluster" then start procedure to import an existing cluster
```

Comment:
```suggestion
1. Create 2 clusters (or more).
2. Choose one cluster as the main one or hub, and install the ACM operator on it.
3. Create a default MultiClusterHub.
4. In the console top bar, select "all cluster" then start the procedure to import an existing cluster.
```

Reply:
A few comments about these steps:

- I think "create a default MultiClusterHub" should be its own step, or a nested step rather than a continuation of the step 3 with a semicolon. 
- Is this a MultiClusterHub custom resource or Operator? I think specifying would be good. 
- I'm wondering, should ACM be spelled out? Or is it an approved acronym that all readers would be familiar with? If its new, I would suggest spelling it out and putting ACM in parentheses for this first mention, then elsewhere you can just use ACM. 
- Periods are needed at the end of these steps.
- I also added some "the".